core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -74,6 +74,22 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
   }
 
   /**
+   * Compute the population standard deviation of this RDD's elements.
+   * @since 2.1.0
```

Comment:
We need to use the Scala `@Since` annotation. Have a look how this is annotated elsewhere in Scala. For Java, yeah, we only use the `@since` javadoc tag because that's all that is available.